lib/modules/dosomething/dosomething_helpers/dosomething_helpers.theme.inc

```diff
@@ -304,9 +304,13 @@ function dosomething_helpers_add_js_campaign_and_user_data($vars) {
  *
  * @return void
  */
-function dosomething_helpers_add_js_onboarding() {
+function dosomething_helpers_add_js_onboarding($nid) {
+  $global_enable = variable_get('dosomething_campaign_enable_onboarding', FALSE);
```

Comment:
`global_enable` sounds like it could be confusing... can we just call it `onboarding_enabled`. Also, nitpicky, but can we either go with `noun_enabled` or `enable_noun`, where _noun_ is either `developer`, `onboarding`, or `campaign`. Again the latter two just sound better when reading it with an associated `true/false` boolean.